scenarios/kubernetes_aws.py

```diff
@@ -95,18 +96,19 @@ def main(args):
     # Enforce to be always present
     aws_ssh = '/workspace/.ssh/kube_aws_rsa'
     aws_pub = '%s.pub' % aws_ssh
-    aws_cred = '/workspace/.ssh/credentials'
+    aws_cred = '/workspace/.aws/credentials'
     service = '/service-account.json'
 
     cmd.extend([
       '-v', '%s:%s:ro' % (args.aws_ssh, aws_ssh),
       '-v', '%s:%s:ro' % (args.aws_pub, aws_pub),
       '-v', '%s:%s:ro' % (args.aws_cred, aws_cred),
-      '-v', '%s:%s:ro' % (args.service_account, service),
       '-e', 'JENKINS_AWS_SSH_PRIVATE_KEY_FILE=%s' % aws_ssh,
       '-e', 'JENKINS_AWS_SSH_PUBLIC_KEY_FILE=%s' % aws_pub,
-      '-e', 'JENKINS_AWS_CREDENTIALS_FILE=%s' % aws_cred,
-      '-e', 'GOOGLE_APPLICATION_CREDENTIALS=%s' % service])
+      '-e', 'JENKINS_AWS_CREDENTIALS_FILE=%s' % aws_cred])
+    if args.service_account:
+      cmd.extend(['-v', '%s:%s:ro' % (args.service_account, service),
```

Comment:
I mean a new line after `([`? (Not 100% sure but I was told this way)

Eh seems lint did not shout so I assume it's fine..